StellarDotnetSdk/Operations/InvokeHostFunctionOperation.cs

```diff
@@ -108,22 +108,24 @@ public class CreateContractOperation : InvokeHostFunctionOperation
     /// <summary>
     ///     Creates a new <c>CreateContractOperation</c> using the provided address and salt.
     /// </summary>
-    /// <param name="wasmId"></param>
+    /// <param name="wasmHash">
+    ///     (aka Wasm ID or hash ID or just hash), a hex-encoded hash of the WebAssembly (Wasm) bytes that represents a compiled smart contract on the Stellar network.
```

Comment:
Just keep it short and but descriptive. I think we don't have to say `on the Stellar network` as we are already in a Stellar SDK so the context is already clear.
```suggestion
    ///     A hex-encoded string of the Wasm bytes of a compiled smart contract.
```

Reply:
Updated.